src/utils/notification-utils.ts

```diff
@@ -8,3 +10,75 @@ export const snackbarSettings: OptionsObject = {
     preventDuplicate: true,
     variant: 'info',
 };
+
```

Comment:
no change needed

But I feel like this is not the right spot for this code. This util is really for notifications displaying in the UI specifically... and the new stuff feels more aligned to the subscriptions and backend implementation